swan-lake/learn/observing-ballerina-code.md

```diff
@@ -310,7 +309,7 @@ The table below provides the descriptions of each configuration option and possi
 Configuration Key | Description | Default Value | Possible Values
 --- | --- | --- | --- 
 b7a.observability.tracing.enabled | Whether tracing is enabled (true) or disabled (false) | false | true or false
-b7a.observability.tracing.name | Tracer name which implements tracer interface. | jaeger | jaeger or zipkin
+b7a.observability.tracing.provider | Tracer name which implements tracer interface. | jaeger | jaeger or if any custom implementation, then name of the tracer.
```

Comment:
```suggestion
b7a.observability.tracing.provider | Tracer name which implements tracer interface. | Jaeger | Jaeger or if any custom implementation, the name of the tracer.
```

Reply:
Name "jaeger" should be all in lowercase letter. Other suggestion fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2